pyfolio/tests/test_tears.py

```diff
@@ -39,11 +39,6 @@ class PositionsTestCase(TestCase):
             __location__ + '/test_data/test_returns.csv.gz'),
         index_col=0, parse_dates=True)
     test_returns = to_series(to_utc(test_returns))
-    test_gross_lev = read_csv(
-        gzip.open(
-            __location__ + '/test_data/test_gross_lev.csv.gz'),
```

Comment:
Great to remove it here, I would, however, add a comparison between inferred gross_lev and what we load from the file.